tox.ini

```diff
@@ -14,7 +15,9 @@ envlist =
 [testenv]
 deps = {[base]testdeps}
 extras = plotly
-commands = pytest {posargs}
+commands =
+    sphinx-build -b doctest -d {toxinidir}/doc/build/doctrees {toxinidir}/doc/source {toxinidir}/doc/build/doctest
```

Comment:
I would keep the sphinx related stuff in pyXX-docs, using `make doctest`.
See for example https://github.com/BlueBrain/atlas-alignment/blob/307aa84ba7ae21be8b9b28bc6f3341226c12867e/tox.ini#L51-L53 or other projects with a similar approach.

Reply:
The issue with that is that it would test the code in the docs only for python3.8 rather than for all supported pythons. This is the main reason for which I think docs and doctest should not be in the same job.

Reply:
I still think that proper unit tests should go under tests/ and tested as usual, while code in the docs should not be part of this job.
It's good to execute the code in the docs to be sure that the documentation is up to date / not broken, but testing with only one version of python is enough IMHO.
If there is code in the docs that's not part of the unit tests, it should be added (and in this way it's considered for the coverage as well).

Reply:
I don't know the original reason, but I guess it can be to keep tox.ini simpler, keep the sphinx logic separated, or for some historical reasons. In general, putting complex commands in tox.ini doesn't work very well.